javalib/src/main/scala/java/util/HashMap.scala

```diff
@@ -0,0 +1,187 @@
+package java.util
+
+import java.{util => ju}
+
+import scala.collection.{mutable => mu}
```

Comment:
This is not very idiomatic. Prefer `import scala.collection.mutable`, and use `mutable.HashMap` for example.